node/src/components/consensus/era_supervisor.rs

```diff
@@ -711,19 +711,19 @@ where
         })
     }
 
-    pub(super) fn handle_block_added(&mut self, block: Block) -> Effects<Event<I>> {
+    pub(super) fn handle_block_added(
+        &mut self,
+        block_hash: BlockHash,
```

Comment:
`BlockHash` isn't necessary here; I am more leery of juggling an invariant than I am of micro-optimizing.

Reply:
Done in https://github.com/CasperLabs/casper-node/pull/1325/commits/e56eeeefffda3e6d09a9a0d7184240c0e872cb92.